### DISCUSSION TITLE
How can I use oh-my-bash along with my own configs

### DISCUSSION THREAD

**Discussion by reteehee** [ohmybash/oh-my-bash#303] [Q&A] [answered]
Is there a way to use oh-my-bash and preserving my own configs. There's some of my configs contrary to oh-my-bash.

My idea is to make my `.bashrc` as `.bahsrc.local` and export it within `.bashrc` (at the last line), so I got expected result when I means do-my-own-things.

Here I just want to make sure, is it OK to add additional definitions in `.bashrc` (template config), is it'll remain there if there's an update?

**Comment by akinomyoga** [comment, marked as answer]
> Here I just want to make sure, is it OK to add additional definitions in `.bashrc` (template config),

Yes.



> is it'll remain there if there's an update?

Yes, it will remain. The update does not change `~/.bashrc` but only changes the contents of `~/.oh-my-bash` (except for `~/.oh-my-bash/custom`).
